test/options/fallback.test.js

```diff
@@ -21,5 +21,64 @@ describe('Options', () => {
 
       expect(source).toMatchSnapshot();
     });
+
+    // Version 1.0.1 passes options provided to url-loader to the fallback as well, so make sure that still works.
+    test('{String}', async () => {
+      const config = {
+        loader: {
+          test: /\.png$/,
+          options: {
+            limit: 100,
+            fallback: 'file-loader',
+            name: '[name].[ext]',
```

Comment:
Let's add tests where name is `[hash].[ext]` to ensure options will send in fallback.